### PR TITLE
Support KONG_ADMIN_TOKEN to be provided via file #99

### PR DESCRIPTION
**Pull request by ludovic-pourrat**
Feature proposal.

Comments and feedback welcome ;-)

### CODE REVIEW

File: src/core/Workspace.ts [hunk 1]
```diff
@@ -44,11 +45,19 @@ export default class Workspace {
       workspace.config.data.kong_admin_url = process.env.KONG_ADMIN_URL
     }
 
-    if (process.env.KONG_ADMIN_TOKEN) {
+    if (process.env.KONG_ADMIN_TOKEN_FILE) {
+      if ((await this.exists(process.env.KONG_ADMIN_TOKEN_FILE)) === false) {
+        throw new Error()
```
Comment (HenriPro) [inline src/core/Workspace.ts:50]: Can we make the error if there is a KONG_ADMIN_TOKEN_FILE, but the file does not exist more verbose?

**Review by HenriPro — CHANGES_REQUESTED** [review]
Besides a request to make the error handling more verbose, this feature seems like a sensible addition, and the implementation seems good.

File: src/core/Workspace.ts (hunk 1 above)
Comment (ludovic-pourrat) [reply src/core/Workspace.ts:52]: @HenriPro see latest push.

**Comment by zekth** [comment]
could you please run `npm run lint:fix`

**Review by HenriPro — APPROVED** [review]

**Comment by HenriPro** [comment]
Thanks for the work on this!

**Pull request merged**